src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/WUXProgressRingTests/UnoSamples_WUXProgressRing_Tests.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using NUnit.Framework;
+using SamplesApp.UITests.TestFramework;
+using Uno.UITest.Helpers;
+using Uno.UITest.Helpers.Queries;
+
+namespace SamplesApp.UITests.Windows_UI_Xaml_Controls.WUXProgressRingTests
+{
+	public class UnoSamples_WUXProgressRing_Tests : SampleControlUITestBase
+	{
+		[Test]
+		[AutoRetry()]
+		[ActivePlatforms(Platform.iOS, Platform.Android)]
+		public void ProgressRing_IsEnabled_Running()
+		{
+			Run("UITests.Windows_UI_Xaml_Controls.ProgressRing.WindowsProgressRing_GH1220");
+
+			var isEnabledTest = _app.Marked("isEnabledTest");
+			var placeHolder = _app.Marked("placeHolder");
+
+			var before = TakeScreenshot("before", new ScreenshotOptions() { IgnoreInSnapshotCompare = true });
+			var placeholderRect = _app.Query(placeHolder).First().Rect;
+			var isEnabledTestRect = _app.Query(isEnabledTest).First().Rect;
+
+			ImageAssert.AreNotEqual(before, placeholderRect, before, isEnabledTestRect, expectedToActualScale: base.GetDisplayScreenScaling());
```

Comment:
I don't really understand the use of `expectedToActualScale` here. `Expected` and `actual` are both from a screenshot here so they should both have the same scale no?

cc @dr1rrb

Reply:
Since we're comparing only a portion of the images, the rectangles provided are provided in logical pixels, not device pixels.

Reply:
Actually I would agree with @davidjohnoliver  (even if it's me who suggested to use it 😳). This is usually useful only when comparing stored assets and actual screenshot.